src/Assets/Asset.php

```diff
@@ -288,6 +288,10 @@ public function absoluteUrl()
 
     public function thumbnailUrl($preset = null)
     {
+        if ($this->isSvg()) {
+            return $this->url();
```

Comment:
When the asset container is private, the `$this->url()` will be `null`, so this will still not work.

You'll need to make that passthrough route I was talking about that just reads the svg contents.

Reply:
I read over that part the first time. Makes sense, I added the route.